### PR TITLE
Add details to PPP

### LINKED ISSUE #191: [PE-D][Tester C] Overzealous error catching
Typing Medium instead of medium throw me an error. Users may be accustomed to typing in Caps and will get frustrated over this. 

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: cadencjk/ped#6

### LINKED ISSUE #199: [PE-D][Tester E] TODOs Same name can be generated with different priority
-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: yixiann/ped#13

### LINKED ISSUE #212: [PE-D][Tester A] Inconsistant of same task.
If you input `todo add d/A date/23-12-2022 pr/low` and again `todo add d/A date/23-12-2022 pr/low`

You will get the following `This Todo already exists in SoConnect!` (Also maybe renaming to task might be better as user might not understand what is todo)

But for some reason if I were to change the priority `todo add d/A date/23-12-2022 pr/medium`, it can be added in. Hence, there can be same task with same name just different priority.

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: LolfoollorS/ped#12

### PR DESCRIPTION
**Pull request by Eugene-Ong-W-X**

### CODE REVIEW

File: docs/team/eugene-ong-w-x.md [hunk 1]
```diff
@@ -5,39 +5,36 @@ title: Eugene Ong Wei Xiang's Project Portfolio Page
...
 
-Given below are my contributions to the project.
+Given below are my contributions to SoConnect.
 
-* **New Feature**: Added the ability to undo/redo previous commands. (remove)
-    * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-    * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-    * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added the ability to sort a list of contacts.
```
Comment (ChongCheeKaiClarence) [inline docs/team/eugene-ong-w-x.md]: I think we are supposed to write the feature itself like `Sort Feature`

File: docs/team/eugene-ong-w-x.md [hunk 2]
```diff
@@ -5,39 +5,36 @@ title: Eugene Ong Wei Xiang's Project Portfolio Page
...
+    * What it does: Allows the user to sort the contact list by a selected parameter. User can sort in reverse as well as combine multiple parameters to sort the list by.
+    * Justification: This feature improves SoConnect's utility as it provides the user freedom in choosing how their contact list is displayed.
+    * Highlights: This enhancement affects the parameters of contacts (e.g. name, address, email, phone, tags). For every parameter stored in a contact, it is necessary to design the sorting order and implement it accordingly. It required an in-depth analysis on how every parameter should be logically sorted. The implementation too was challenging as it required changes to multiple levels of the model component of SoConnect.
 
-* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=eugene-ong-w-x&breakdown=true)
+* **New Features**: Added the ability to add, remove, edit, and filter tasks (referred to as todos).
```
Comment (ChongCheeKaiClarence) [inline docs/team/eugene-ong-w-x.md]: Same thing

File: docs/team/eugene-ong-w-x.md [hunk 3]
```diff
@@ -5,39 +5,36 @@ title: Eugene Ong Wei Xiang's Project Portfolio Page
...
+    * What it does: Allows the user to manage todos in addition to contacts. Basic management features such as adding, removing, editing todos are available. Users can also filter the todos displayed.
+    * Justification: This feature significantly improves SoConnect's utility as users can now use it to manage both contacts and tasks, making SoConnect multipurpose.
+    * Highlights: This enhancement affects the design of SoConnect as it now stores data of todos in addition to data of contacts. It required in-depth analysis on design alternatives. The implementation too was tedious as it required heavy modifications to the model, storage, and logic components.
```
Comment (ChongCheeKaiClarence) [inline docs/team/eugene-ong-w-x.md]: I think you can remove the  `too`

File: docs/team/eugene-ong-w-x.md [hunk 4]
```diff
@@ -5,39 +5,36 @@ title: Eugene Ong Wei Xiang's Project Portfolio Page
...
-    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added the ability to sort a list of contacts.
+    * What it does: Allows the user to sort the contact list by a selected parameter. User can sort in reverse as well as combine multiple parameters to sort the list by.
+    * Justification: This feature improves SoConnect's utility as it provides the user freedom in choosing how their contact list is displayed.
+    * Highlights: This enhancement affects the parameters of contacts (e.g. name, address, email, phone, tags). For every parameter stored in a contact, it is necessary to design the sorting order and implement it accordingly. It required an in-depth analysis on how every parameter should be logically sorted. The implementation too was challenging as it required changes to multiple levels of the model component of SoConnect.
```
Comment (ChongCheeKaiClarence) [inline docs/team/eugene-ong-w-x.md]: I think you can remove the `too`

**Review by ChongCheeKaiClarence — COMMENTED** [review]
Just some nits, otherwise, LGTM

File: docs/team/eugene-ong-w-x.md (hunk 4 above)
Comment (Eugene-Ong-W-X) [reply docs/team/eugene-ong-w-x.md]: Ok

File: docs/team/eugene-ong-w-x.md (hunk 3 above)
Comment (Eugene-Ong-W-X) [reply docs/team/eugene-ong-w-x.md]: Ok

File: docs/team/eugene-ong-w-x.md (hunk 2 above)
Comment (Eugene-Ong-W-X) [reply docs/team/eugene-ong-w-x.md]: Ok

File: docs/team/eugene-ong-w-x.md (hunk 1 above)
Comment (Eugene-Ong-W-X) [reply docs/team/eugene-ong-w-x.md]: Ok good idea

Comment (Eugene-Ong-W-X) [reply docs/team/eugene-ong-w-x.md:12]: Ok good idea

**Pull request merged**